<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clientwidget</class>
 <widget class="QWidget" name="clientwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1644</width>
    <height>894</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>90</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>服務器端口</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>40</y>
     <width>74</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>服務器ip</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_send">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>450</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>send</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ip">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>40</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>192.168.1.40</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_port">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>9</y>
     <width>142</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>4567</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>接收數據</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>60</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>發送數據</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_recv">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>251</width>
     <height>131</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_send">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>251</width>
     <height>177</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_connect">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>connect</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_card_1">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>201</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_card_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>10</y>
     <width>201</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_card_3">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>10</y>
     <width>201</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_card_4">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>10</y>
     <width>201</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_card_7">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>580</y>
     <width>201</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_card_6">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>300</y>
     <width>201</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_card_5">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>300</y>
     <width>201</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_card_8">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>580</y>
     <width>201</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_card_9">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>580</y>
     <width>201</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_card_10">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>580</y>
     <width>201</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="deck_1">
   <property name="geometry">
    <rect>
     <x>1390</x>
     <y>10</y>
     <width>201</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="deck_2">
   <property name="geometry">
    <rect>
     <x>1390</x>
     <y>580</y>
     <width>201</width>
     <height>281</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_recv_3">
   <property name="geometry">
    <rect>
     <x>1410</x>
     <y>550</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_recv_2">
   <property name="geometry">
    <rect>
     <x>1410</x>
     <y>290</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_recv_4">
   <property name="geometry">
    <rect>
     <x>1350</x>
     <y>390</y>
     <width>261</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:20pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:400; color:#ef2929;&quot;&gt;start&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>pushButton_send</zorder>
  <zorder>pushButton_close</zorder>
  <zorder>lineEdit_port</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>textEdit_recv</zorder>
  <zorder>textEdit_send</zorder>
  <zorder>lineEdit_ip</zorder>
  <zorder>pushButton_connect</zorder>
  <zorder>label_card_1</zorder>
  <zorder>label_card_2</zorder>
  <zorder>label_card_3</zorder>
  <zorder>label_card_4</zorder>
  <zorder>label_card_7</zorder>
  <zorder>label_card_6</zorder>
  <zorder>label_card_5</zorder>
  <zorder>label_card_8</zorder>
  <zorder>label_card_9</zorder>
  <zorder>label_card_10</zorder>
  <zorder>deck_1</zorder>
  <zorder>deck_2</zorder>
  <zorder>textEdit_recv_3</zorder>
  <zorder>textEdit_recv_2</zorder>
  <zorder>textEdit_recv_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
